web_src/js/utils/dom.ts

```diff
@@ -302,8 +302,15 @@ export function replaceTextareaSelection(textarea: HTMLTextAreaElement, text: st
 
 // Warning: Do not enter any unsanitized variables here
 export function createElementFromHTML(htmlString: string): HTMLElement {
```

Comment:
```suggestion
export function createElementFromHTML<T = HTMLElement>(htmlString: string): T {
```

and change the return types to `T`. This will allow the user to call the function with `createElementFromHTML<HTMLDivElement>` for example.

Reply:
But we never use the `HTMLDivElement` type in code. The default `HTMLElement` is good enough

Reply:
But it might be useful in the future being able to specificy exact return type.

Reply:
`createElementFromHTML<T extends HTMLElement>` should be enough

Reply:
Maybe, but see below.